forc/src/utils/restricted_names.rs

```diff
@@ -93,36 +94,34 @@ pub fn is_glob_pattern<T: AsRef<str>>(name: T) -> bool {
     name.as_ref().contains(&['*', '?', '[', ']'][..])
 }
 
-#[test]
-fn test_invalid_char() {
-    assert_eq!(
-        contains_invalid_char("test#proj", "package name"),
```

Comment:
You should be able to get this to work again by just mapping the error returned by anyhow back to a string

`contains_invalid_char("test#proj", "package name").map_err(|e| e.to_string())`